src/pages/ProfileForm/CreateProfileForm/index.tsx

```diff
@@ -105,7 +105,7 @@ function CreateProfileForm(): JSX.Element {
   return (
     <div id='create-profile-form-no-associations'>
       <CrumbTrail>
-        <Link href='/'>Platform home</Link>
+        <Link href='/landing'>Platform home</Link>
```

Comment:
Heyo! I don't think we want `/landing` on this form, but want to keep it `/`. Otherwise, when you click it, since their user hasn't been associated with any institution yet, the'll be redirected to the same page.